src/backend/distributed/commands/collation.c

```diff
@@ -320,6 +321,15 @@ PostprocessAlterCollationOwnerStmt(Node *node, const char *queryString)
 		return NIL;
 	}
 
+	bool createInProgress = true;
+	DeferredErrorMessage *errMsg = DeferErrorIfHasUnsupportedDependency(&collationAddress,
+																		createInProgress);
```

Comment:
I don't think we need this on ALTER COLLATION. The collation is already distributed.